quint/src/quintVerifier.ts

```diff
@@ -142,30 +142,33 @@ function err<A>(explanation: string, errors: ErrorMessage[] = [], traces?: ItfTr
 }
 
 function findApalacheDistribution(): VerifyResult<ApalacheDist> {
-  const configuredDist =
-    process.env.APALACHE_DIST && path.isAbsolute(process.env.APALACHE_DIST)
-      ? process.env.APALACHE_DIST
-      : path.join(process.cwd(), process.env.APALACHE_DIST!)
-
-  // TODO: fetch release if APALACHE_DIST is not configured
-  // See https://github.com/informalsystems/quint/issues/701
-  let distResult: VerifyResult<string> = err(
-    'Unable to find the apalache distribution. Ensure the APALACHE_DIST enviroment variable is set.'
-  )
-
-  if (configuredDist && !fs.existsSync(configuredDist)) {
-    distResult = err(`Specified APALACHE_DIST ${configuredDist} does not exist`)
-  } else if (configuredDist) {
-    distResult = right(configuredDist)
+  if (!process.env.APALACHE_DIST) {
+    return err('APALACHE_DIST enviroment variable is not set.')
   }
 
-  return distResult.chain(dist => {
-    const jar = path.join(dist, 'lib', 'apalache.jar')
-    const exe = path.join(dist, 'bin', 'apalache-mc')
-    return fs.existsSync(jar) && fs.existsSync(exe)
-      ? right({ jar, exe })
-      : err(`Apalache distribution is corrupted. Cannot find ${jar} or ${exe}.`)
-  })
+  const dist = path.isAbsolute(process.env.APALACHE_DIST)
+    ? process.env.APALACHE_DIST
+    : path.join(process.cwd(), process.env.APALACHE_DIST!)
+
+  if (!fs.existsSync(dist)) {
+    return err(`Specified APALACHE_DIST ${dist} does not exist.`)
+  }
+
+  const jar = path.join(dist, 'lib', 'apalache.jar')
+  const exe = path.join(dist, 'bin', 'apalache-mc')
+
+  if (!fs.existsSync(jar)) {
+    return err(
+      `Apalache distribution is corrupted: cannot find ${jar}. Ensure the APALACHE_DIST environment variable is set.`
```

Comment:
iiuc, we should already know at this point whether the envvar is set:

```suggestion
      `Apalache distribution is corrupted: cannot find ${jar}.`
```

Reply:
Good catch! Meant to rephrase the second sentence: 739ced81